client/src/App.tsx

```diff
@@ -29,6 +29,7 @@ function App(): JSX.Element {
                 <Route exact path="/signup" component={Signup} />
                 <AuthRoute exact path="/dashboard" component={Dashboard} />
                 <AuthRoute path="/profile/settings" component={Settings} />
+                <Route path="/profileLists" component={ProfileLists} />
```

Comment:
```suggestion
                <Route path="/profile-listings" component={ProfileLists} />
```
Hyphenated case is the standard naming convention for route names in JS 😄

Reply:
changed the route path name.